service-registry/src/main/java/io/servicecomb/serviceregistry/consumer/MicroserviceVersions.java

```diff
@@ -45,6 +46,8 @@
 
   private String microserviceName;
 
+  private String revision = DefinitionConst.DEFAULT_REVISION;
```

Comment:
default must be null
if did not get revison from SC, then in request header must not exist revision key.

Reply:
done